static/app/components/charts/utils.tsx

```diff
@@ -185,10 +185,9 @@ export function shouldFetchPreviousPeriod({
  * Generates a series selection based on the query parameters defined by the location.
  */
 export function getSeriesSelection(
-  location: Location,
-  parameter = 'unselectedSeries'
+  location: Location
 ): LegendComponentOption['selected'] {
-  const unselectedSeries = decodeList(location?.query[parameter]);
+  const unselectedSeries = decodeList(location?.query.unselectedSeries);
```

Comment:
Nit: assuming `location` is a required field, the `?` isn't necessary here.

```suggestion
  const unselectedSeries = decodeList(location.query.unselectedSeries);
```

Reply:
done!